app/src/main/java/com/duckduckgo/app/brokensite/BrokenSiteViewModel.kt

```diff
@@ -103,7 +105,15 @@ class BrokenSiteViewModel(
 
     fun onSubmitPressed(webViewVersion: String) {
         if (url.isNotEmpty()) {
-            val brokenSite = getBrokenSite(webViewVersion)
+
+            val lastTrackingInfo = trackingLinkDetector.lastTrackingLinkInfo
```

Comment:
I assume this is a known edge case. But since this is a singleton, this value could be change from another consumer.

Reply:
That is a potential edge case, I've created a follow-up task to remove the `Singleton` annotation from `TrackingLinkDetector`: https://app.asana.com/0/414730916066338/1201750076224001/f